frontend/web/components/Feature.js

```diff
@@ -102,13 +123,30 @@ export default class Feature extends PureComponent {
               }
               tooltip={`${Constants.strings.REMOTE_CONFIG_DESCRIPTION}${
                 !isEdit
-                  ? '<br/>Setting this when creating a feature will set the value for all environments. You can edit the this individually for each environment once the feature is created.'
+                  ? '<br/>Setting this when creating a feature will set the value for all environments. You can edit this individually for each environment once the feature is created.'
                   : ''
               }`}
               title={`${valueString}`}
             />
           </FormGroup>
         )}
+        {this.state.isNegativeNumberString && (
+          <WarningMessage
+            warningMessage={
+              <div>
+                This feature currently has the value of{' '}
+                <strong>"{environmentFlag?.feature_state_value}"</strong>.
+                Saving this feature will convert its value from a string to a
+                number. If you wish to preserve this value as a string, please
+                save it via our{' '}
+                <a href='https://api.flagsmith.com/api/v1/docs/#/api/api_v1_environments_featurestates_update'>
```

Comment:
Minor nit - suggest wording this as `please save it using the Admin API` for consistency with other docs. LGTM, good stuff 🙏